bin/p2-dsctl/main.go

```diff
@@ -194,6 +201,14 @@ func main() {
 					ds.PodID = podID
 				}
 			}
+			// If the timeout is not specified, the default timeout is time.Duration(0)
+			// We don't want a timeout to be 0 anyways because nothing will be scheduled
+			if *updateTimeout != time.Duration(0) {
```

Comment:
if createTimeout is 0, you exit the program but if updateTimeout is 0, you just don't update it. Probably exit here too?